### DISCUSSION TITLE
How is svg-sprite-loader configured in rsbuild or how to import svg in vue like svg-sprite-loader can change svg color

### DISCUSSION THREAD

**Discussion by pengpengliang** [web-infra-dev/rsbuild#3558] [Q&A]
I know  @rsbuild/plugin-svgr but it seems need React enviroment .

**Comment by 9aoy** [comment]
You can add a custom rule and `svg-sprite-loader` just like in webpack.

https://rsbuild.dev/guide/basic/static-assets#custom-rules

**Reply by pengpengliang** [reply]
![2](https://github.com/user-attachments/assets/72e8755e-e4f7-4607-aa3e-f73a5b3f29f0)
![1](https://github.com/user-attachments/assets/aae595bb-b043-4bc6-8de5-d181ac1402fa)
it doesn't work. svg element unable to inject body. Is what I wrote wrong?
